c-cannons/bangazon-initial_site
# Testing images

#### Descriptions

Updated seed file and one migration file

#### Steps to Test

1. Delete current database.
2. ```rails db:migrate```
3. ```rails db:seed```
Data should seed without having to create 4 customers first
